formspree/forms/views.py

```diff
@@ -150,8 +155,23 @@ def send(email_or_string):
         email = email_or_string.lower()
 
         # get the form for this request
-        form = Form.query.filter_by(hash=HASH(email, host)).first() \
-               or Form(email, host) # or create it if it doesn't exists
+        form = Form.query.filter_by(hash=HASH(email, host)).first()
+
+        # or create it if it doesn't exist
+        if not form:
+            if not url_domain(settings.SERVICE_URL) in host:
+                form = Form(email, host)
+            else:
+                # Bad user is trying to submit a form spoofing formspree.io
+                # Error out silently
+                if request_wants_json():
+                    return jsonerror(400, {'error': "Unable to submit form"})
+                else:
+                    return render_template(
+                        'error.html',
+                        title='Unable to submit form',
+                        text='Sorry'), 400
+
```

Comment:
It seems to me that this will allow for the creation of forms from AJAX as long as it doesn't contain `SERVICE_URL` in the host.

Reply:
Yeah this doesn't have anything to do with AJAX. We're just trying to keep people from submitting forms by faking the referrer to look like it's coming from formspree.io. Users get confused and sometimes accept these confirmation emails because they trust formspree.

Reply:
I guess they'd still need to use AJAX to automate these emails, to bypass captcha. Regardless there's a separate PR for that.